README.md

```diff
@@ -112,12 +112,14 @@ Then using our Addons component in ChatScreen
 ```javascript
 import React from 'react'
 import {ChatScreen as BaseChatScreen} from 'rn-firebase-chat'
-import {CameraView, useCamera} from 'rn-firebase-chat/src/addons/camera'
+import {CameraView, useCamera, FileAttachmentRef, FileAttachment} from 'rn-firebase-chat/src/addons/camera'
```

Comment:
I think we should separate to `rn-firebase-chat/src/addons/file`

Reply:
I updated